packages/payload/src/collections/config/types.ts

```diff
@@ -525,11 +531,18 @@ export interface SanitizedCollectionConfig
   auth: Auth
   endpoints: Endpoint[] | false
   fields: Field[]
-  slug: CollectionSlug
+
+  /**
+   * Fields in the database schema structure
+   * Rows / collapsible / tabs w/o name `fields` merged to top, UIs are excluded
+   */
+  flattenFields: FlattenField[]
```

Comment:
I think this should be named `flattenedFields` to be more clear. `flattenFields` sounds like a function.